pkg/collect/host_run.go

```diff
@@ -12,9 +14,12 @@ import (
 )
 
 type HostRunInfo struct {
-	Command  string `json:"command"`
-	ExitCode string `json:"exitCode"`
-	Error    string `json:"error"`
+	Command   string   `json:"command"`
+	ExitCode  string   `json:"exitCode"`
+	Error     string   `json:"error"`
+	OutputDir string   `json:"outputDir"`
+	Input     string   `json:"input"`
```

Comment:
Should this be a `map[string]string` or `map[string]any`? I could not load this yaml. How is the `input` parameter meant to behave? Isn't the `key` meant to be the filename and the the `value` be the contents of the file, all stored as flat files in `TS_INPUT` directory?

```yaml
        input:
          dummy.yaml: My sample data REDACTED SECRET
          config:   # I cannot load this config
            dummy.yaml: |-
              username: postgres
              password: <my-pass>
            config.toml: |-
              title = "TOML Example"
              [owner]
              name = "Tom Preston-Werner"
```

Reply:
So the input files will be written into the `TS_INPUT_DIR`, which is a temp directory. It will always be deleted. I previously also choose to output the input file to somewhere but then [removed that part](https://github.com/replicatedhq/troubleshoot/pull/1288/commits/08714e06d00efdd6f72e271720c2ac2478a30c30) -- talked with Martin and we thought input might be sensitive and might be good to just skip storing it.

The user triggers this bundle will have input file on their hand anyways but anything output to that bundle might do harm when the bundle is given to someone else.

Reply:
or btw it should be

```yaml
        input:
          dummy.yaml: My sample data REDACTED SECRET
          dummy2.yaml: |-
            username: postgres
            password: <my-pass>
          config.toml: |-
            title = "TOML Example"
            [owner]
            name = "Tom Preston-Werner"
```

that's why its `map[string]string`